test/Assertions.php

```diff
@@ -21,6 +24,13 @@
 
 trait Assertions
 {
+    public static function getObjectPropertyHydratorClass() : string
+    {
+        return class_exists(ObjectPropertyHydrator::class)
```

Comment:
This can be dropped if we switch to `^3.0` only.

Reply:
See comment above :)